### PR TITLE
Fixes #19628: Update technique to use RUDDER_REPORTS_DB_URL and RUDDER_REPORTS_DB_PASSWORD

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/19628

### CODE REVIEW

**Comment by fanf** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
LGTM, can be merged when the webapp part is merged. I'll do the follow-up PR to actually use the new values.

**Comment by fanf** [comment]
@amousset : marked to merge, you can prepare relayd changes

File: techniques/system/rudder-service-postgresql/1.0/main.st [hunk 1]
```diff
@@ -27,8 +27,8 @@ bundle common rudder_postgresql {
       "no"                 int => getfields("RUDDER_PSQL_PASSWORD:.*","/opt/rudder/etc/rudder-passwords.conf",":","password_getter");
       "postgres_passwd" string => "${password_getter[2]}";
 
-      "db_name"         string => "&RUDDER_REPORTS_DB_NAME&";
```
Comment (ncharles) [inline techniques/system/rudder-service-postgresql/1.0/main.st:30]: this was necessary for .pgpass

**Pull request merged**